allium/templates/contact.html

```
@@ -101,6 +101,33 @@ <h4 style="margin-top: 0; color: #495057;">📋 Contact & Network Overview</h4>
+                       resolved here BUT some of their relays out there
+                       publish a `url:<this-domain>` token without
+                       declaring `ciissversion:` (or with a mismatched
+                       version/proof pair), categorisation flagged them
+                       as 'incomplete sibling relays'. They appear on
+                       leaderboards as a separate, fragmented entry
+                       (now shown as '<domain> (incomplete AROI)' per
+                       Option A). Surface the count here so this
+                       operator is told what to fix in the misconfigured
+                       relays' ContactInfo. #}
+                    {%- set _sibling_count = (contact_display_data.get('incomplete_sibling_count', 0) if contact_display_data else 0) -%}
+                    {%- if _sibling_count > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)